### ISSUE TITLE
Gradle Error While Compiling

### ISSUE THREAD

**Issue by jaydjohno** [woocommerce/woocommerce-android#3059]
Hey All

I installed Android Studio and cloned the Github project into my documents, i then opened it in Android Studio, and installed all my pre-requisites and everything seems fine.

Now when it tries to build the project i get this error message.

A problem occurred evaluating root project 'woocommerce-android'.
> A problem occurred configuring project ':WooCommerce'.
   > Ambiguous method overloading for method java.io.File#<init>.
     Cannot resolve which method to invoke for [null, class java.lang.String] due to overlapping prototypes between:
     	[class java.lang.String, class java.lang.String]
     	[class java.io.File, class java.lang.String]

I have tried to set the SDK location and this is all correct but still get this error and i dont know why, only thing i did was add the secret in as part of the instructions, not modified anything.

**Comment by hafizrahman** [comment]
I ran into this issue as well. 

My setup is:
- Android Studio 4.1.2
- MacOS Big Sur 11.0.1

This seems to be related to https://stackoverflow.com/questions/64641632/error-updating-android-studio-gradle-from-4-0-2-to-4-1-0 , which points to a fix in:

https://github.com/getsentry/sentry-android-gradle-plugin/issues/35

However the fix was to update the sentry grade plugin's version. Mine is now like this:

`classpath 'io.sentry:sentry-android-gradle-plugin:1.7.36` (in gradle.properties in `/WooCommerce`)

So for the moment my temporary fix is to have that change added locally to my machine. It's curious how it works on another machine of mine, but I got this issue on my main computer, and I've yet to figure out what's really the difference between the two 🤔
